support-frontend/app/admin/settings/Settings.scala

```diff
@@ -101,7 +101,13 @@ object SettingsSource extends LazyLogging {
 
 }
 
-case class PaymentMethodsSwitch(stripe: SwitchState, payPal: SwitchState, directDebit: Option[SwitchState])
+case class PaymentMethodsSwitch(
+  stripe: SwitchState,
+  payPal: SwitchState,
+  directDebit: Option[SwitchState],
+  existingCard: Option[SwitchState],
```

Comment:
What benefit do we get from having two separate switches here? 

If I recall correctly the main purpose of these switches is that it allows us to disable a payment method if the third-party is down (e.g. we can hide Stripe completely if we know that Stripe Checkout will fail to load). For the reuse of existing payment methods we don't need to talk to third-parties client side, so I think a single switch could be sufficient?

Reply:
Given out change of approach in `support-workers` to clone payment methods (due to concerns around auto-pay, see https://github.com/guardian/members-data-api/pull/379#issuecomment-476304459) we will actually have to call 3rd parties for existing due to limitations in Zuora, so being able to switch them separately is definitely necessary.

The original rationale was more to do with being able to push people in one direction and in-case we discovered issues in production say with re-using Direct Debit being able to switch it off without disabling the whole feature... which is still valid.

Reply:
Yep agreed, thanks for explaining 👍